code/controllers/subsystem/chat.dm

```diff
@@ -11,8 +11,11 @@ SUBSYSTEM_DEF(chat)
 /datum/controller/subsystem/chat/fire()
 	for(var/i in payload)
 		var/client/C = i
-		C << output(payload[C], "browseroutput:output")
-		payload -= C
+		if(C)
+			C << output(payload[C], "browseroutput:output")
+			payload -= C
+		else
+			qdel(i)
```

Comment:
Wait a minute, this would only be reached if `C` is `null`, and so `i` would be `null`, and so this would always be `qdel(null)`. Instead, I would advocate that you do `payload -= i`
(Granted, I'm not sure how matching keys in an associated list would work - I think the game does a hard-del when a client disconnects either way)

Reply:
Yeah you're right